libs/workordersidesheet/src/lib/ui-sidesheet/WorkorderSidesheet.tsx

```diff
@@ -73,6 +87,28 @@ export const WorkorderSidesheet = (props: {
     }
   );
 
+  const mccrIcon = (status: string) => {
+    return <h3>{status}</h3>;
+  };
+
+  const tagsOverlay = useMemo(() => {
+    return modelConfig?.tags?.map((tag) => ({
+      tagNo: tag.tagNo,
+      description: tag.description,
+      status: tag.status,
+      icon: mccrIcon(tag.status || ''),
+    })) as TagOverlay[];
+  }, [mccr]);
+
+  const viewerOptions = {
+    statusResolver: (status: string) => {
+      return hasProperty(colorMap, status)
+        ? colorMap[status as PackageStatus]
+        : '#009922';
+    },
+    defaultCroppingDistance: 3,
+  };
+

```

Comment:
Can be extracted outside the component scope

Reply:
Moved this outside the component